quality/postman/sor-1-_table.postman_environment.json

```diff
@@ -0,0 +1,49 @@
+{
+	"id": "191023d9-a402-493b-a690-9bcee390571e",
+	"name": "/sor/1/_table/:table",
+	"values": [
+		{
+			"key": "api-key",
+			"value": "local_admin",
+			"enabled": true
+		},
+		{
+			"key": "api-key-no-rights",
+			"value": "should be passed as param in command line",
+			"enabled": true
+		},
+		{
+			"key": "baseUrl",
```

Comment:
as we have `baseUrl_EU` and `baseUrl_US`, do we really need `baseUrl` parameter?

Reply:
baseUrl has been added along with  _US & _EU just to have it clearer on jenkins side when we need to pass url param explicitly in CLI. Now there it looks like newman run "collection name" -e "environment file" --env-var "baseUrl=%url to emodb%" 
To me, If we had only _US & _EU postfixed url it would look like why we run it always US/EU side.

Reply:
as we don't have `eu` and `us` data centers on the local machine, i would rather introduce `base_url_dc1` and `base_url_dc2` variables